.github/ISSUE_TEMPLATE/page-request.yml

```
@@ -25,16 +23,7 @@ body:
         - Android
         - macOS
         - Windows
-        - General
         - Other
+        - More than one operating system
```

```suggestion
        - More than one operating system
```

What about to use `multiple: true` instead of adding the new choice?

---

Yep, I second the idea of check-boxes :)

---

But it seems a little confusing when you have both. So, when you check `More than one operating system`, what should you select in the dropdown menu?

---

@marchersimon, I meant to replace the dropdown with checkboxes.

---

Oh, my bad. It is just often not easy to find out on which platforms a command works exactly, just that it works on more than one. For example, take `git`. It definitely works on `linux`, `osx`, and `windows`. But does it work on `sunos` and `android`? And is it important to know that? It would go into the `common` platform anyways. 

So I think we should include the `common` platform as an option. (Even tough I prefer radio boxes to a dropdown menu).

---

We can't list all possible platforms. So yeah, it's nice idea to include `common` platform for commands working on all listed platforms.